app/graphql/mutations/create_video.rb

```diff
@@ -0,0 +1,28 @@
+module Mutations
+  class CreateVideo < BaseMutation
+    argument :vimeo_id, Integer, required: true
+    argument :speaker, String, required: false
+    argument :topic, String, required: false
+    argument :date_recorded, String, required: false
+
+    type Types::VideoType
+
+    def resolve(vimeo_id: nil, speaker: nil, topic: nil, date_recorded: nil)
+      video = VideoFacade.new(vimeo_id)
+      Video.create!(
+        title: video.title,
+        description: video.description,
+        speaker: speaker,
+        topic: topic,
+        length: video.length,
+        date_recorded: date_recorded,
```

Comment:
At first, seeing the `speaker`, `topic`, and `date_recorded` attributes being assigned this way while the rest of the attributes are first fed through the facade felt like a code smell to me. I dug a little deeper and I think it makes sense-- all the other attributes are pulled off the facade object since they're retrieved using an API call in VideoService.
For dev empathy, my suggestion would be to reorder these arguments and group similar ones together (i.e. vimeo_id, speaker, topic, and date_recorded would either be at the top or the bottom of the list).